ts/src/handlers.ts

```diff
@@ -38,7 +38,7 @@ enum ExchangeMethods {
     MarketSellOrdersNoThrow = 'marketSellOrdersNoThrow',
     MarketBuyOrders = 'marketBuyOrders',
     MarketBuyOrdersNoThrow = 'marketBuyOrdersNoThrow',
-
+    MatchOrders = 'matchOrders',
```

Comment:
Why did you add this one?

Reply:
Oooops. I think this is leftover from something :/ I didn't mean to, will remove it.